RebuildWithLocalMSBuild.cmd

```diff
@@ -1,13 +1,68 @@
 :: This script will:
-::   1) Rebuild MSBuild source tree.
+::   1) Build MSBuild source tree.
```

Comment:
I'd leave this as "rebuild" because we call the Rebuild (=`Clean;Build`) target.